### PR TITLE
nested .ini files

### PR DESCRIPTION
**Pull request by randaz81**
`yarp::os::Property` class now implements the new tag `import <contextname> <filename>` which allows to include .ini configuration files belonging to different contexts

### CODE REVIEW

**Comment by randaz81** [comment]
This PR fixes: https://github.com/robotology/yarp/issues/565

File: tests/conf/contexts/context2/c2f2.ini [hunk 1]
```diff
@@ -0,0 +1,6 @@
+c2f2p1 15
+c2f2p2 16
+
+[c2f2s1]
+c2f2s1p1 17
+c2f2s1p2 18
```
Comment (drdanz) [inline tests/conf/contexts/context2/c2f2.ini:6]:
```suggestion
c2f2s1p2 18

```

**Review by drdanz — APPROVED** [review]
LGTM, but we should spend some more time in investigating the order of include/import, and values replacements

**Comment by randaz81** [comment]
> LGTM, but we should spend some more time in investigating the order of include/import, and values replacements

I put some relevant info here: https://github.com/robotology/yarp/discussions/2453

**Pull request merged**